meetings/2017-07-06.md

```diff
@@ -0,0 +1,98 @@
+
+### nodejs/community-committee
+
+* Adding Jenn, Tierney, Michael, Olivia, and Richard as members [#77](https://github.com/nodejs/community-committee/pull/77)
+
+### nodejs/TSC
+
+* meta: moderation policy updates [#276](https://github.com/nodejs/TSC/pull/276)
+* Candidate "areas" for TSC and Community Committee  [#278](https://github.com/nodejs/TSC/issues/278)
+
+
+## Minutes
+
+### Adding Jenn, Tierney, Michael, Olivia, and Richard as members [#77](https://github.com/nodejs/community-committee/pull/77)
+
+**In-meeting votes:**
+* Tracy: +1
+* William: +1
+* Bryan: +1
+* Gregor: +1
+
+Voting will have to continue in the repo, in GitHub. Tracy [requested](https://github.com/nodejs/community-committee/pull/77#issuecomment-313516728) that members not present in the meeting vote.
+ 
+Discussion around 3 month limit on member inactivity, William clarified that this landed recently and is not applied retroactively.
+
+**Action Items:**
+  * Reach out to inactive members to see if they’re still interested in participating
+  * Reach out to all members about voting on the additional members
+
+### meta: moderation policy updates [#276](https://github.com/nodejs/TSC/pull/276)
+Request that CommComm review this issue, as it needs approval by the CommComm. Bryan noted that there will likely be a vote at some point, and requested that the CommComm members be informed about the issue.
+
+Possibly have a joint meeting with the TSC to have a discussion on the issue.
+
+**Action Items:**
+  * Bryan to work on getting org access
+  * For everyone: Review this issue
+
+### Candidate "areas" for TSC and Community Committee  [#278](https://github.com/nodejs/TSC/issues/278)
+
+This issue is a general awareness and discussion starter for the CommComm. Michael started the discussion around this.
+
+Tracy brought up the idea of one joint repo for TSC/CommComm discussions, to enable one thread for discussions that apply to both the TSC and CommComm.
+
+Gregor requested further context on travel funds, and Tracy shared the context of what the travel fund is and the need for collaboration across the TSC/CommComm
+
+The group discussed different possibilities for enabling cross-Committee collaboration. Some possibilities are nodejs/collaborators and an admin team or repo.
+
+Further discussion around this issue came up again later in the meeting. The group talked about how to contribute and add to the list, in addition to adding a bit more context to this issue.
+
+One topic the group talked about as a part of this was the new iteration of the Node.js website and the inactivity in the project. The group discussed some of the history surrounding this new iteration, and what possible next steps may be including opening an issue in the Website WG repo to discuss how to proceed and the possibility of getting funding from the foundation to help with associated costs.
+
+**Action Items:**
+  * Create an issue around enabling cross-Committee discussion and collaboration
```

Comment:
IIRC, @hackygolucky is taking this action item on, correct? Can we add that info here?